File: tests/julia/hello.jl
#!/usr/bin/env julia
## this a comment
#= And here comes a multiline comment
   which can be written over several lines 
   it is ended with an equal sign anfd and
   hash sign like here: =#
println("hello world")

## loop example
for i in 1:10
    if i % 3 == 0
        continue
    end
    println(i)
end 

    

File: tests/nim/hello.nim
#!/usr/bin/env nim r
# comment

#[
This is a multiline comment
which can span over several lines
]#

echo "Hello Nim World!"

var x = 2
echo x
if x > 3:
  echo "x is large"
else:
  echo "x is small"
#echo y
proc hello*(h: string): string =
  "Hello " & h & "!"

echo hello("Max")
#echo "What is you name? "
#var name: string = readLine(stdin)

#echo name
